public/js/actions/pause.js

```diff
@@ -83,6 +89,22 @@ function command({ type }) {
   };
 }
 
+function stepIn() {
+  return command({ type: "stepIn" });
+}
+
+function stepOver() {
+  return command({ type: "stepOver" });
+}
+
+function stepOut() {
+  return command({ type: "stepOut" });
+}
+
+function resume() {
+  return command({ type: "resume" });
+}
```

Comment:
these actions used to be called through `command`, but calling them directly makes the code more readable/understandable.